web/src/DetailedInfoPage.tsx

```diff
@@ -372,6 +391,133 @@ export const DetailedInfoPage: React.FunctionComponent<{}> = () => {
         });
     };
 
+    const renderSeqSection = () => {
+        if (!sample || !sequenceInfo) {
+            return <></>;
+        }
+        return (
+            <>
+                <h4
+                    style={{
+                        borderBottom: `1px solid black`,
+                    }}
+                >
+                    Sequence Information
+                </h4>
+                {renderSeqInfo(sequenceInfo)}
+                {prepReadMetadata(sequenceInfo.meta || {})}
+            </>
+        );
+    };
+
+    const renderSampleSection = () => {
+        if (!sample || !sampleInfo) {
+            return <></>;
+        }
+        return (
+            <>
+                <h4
+                    style={{
+                        borderBottom: `1px solid black`,
+                    }}
+                >
+                    Sample Information
+                </h4>
+                {Object.entries(sampleInfo)
+                    .filter(([key, value]) =>
+                        sampleFieldsToDisplay.includes(key)
+                    )
+                    .map(([key, value]) =>
+                        value ? (
+                            <div key={`${key}-${value}`}>
+                                <b>{key}:</b> {value.toString()}
+                            </div>
+                        ) : (
+                            <React.Fragment
+                                key={`${key}-${value}`}
+                            ></React.Fragment>
+                        )
+                    )}
+            </>
+        );
+    };
+
+    const renderPedigreeSection = () => {
```

Comment:
Can do this later, but would be good to make this it's own separate component importable in other places.

Reply:
Might do in future - haven't done yet as might be swapping to a separate package at some point